src/api-explorer/v4-0/ConcurRequest.swagger2.json

```diff
@@ -2282,9 +2286,491 @@
+                  {
+                    "errorCode": "userNotFound",
+                    "errorMessage": "User is not found"
+                  }
+                ]
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "403": {
+            "description": "You do not have permission to perform this operation.",
+            "examples": {
+              "application/json": {
+                "errors": [
+                  {
+                    "errorCode": "permissionDenied",
+                    "errorMessage": "Permission denied"
+                  }
+                ]
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "500": {
+            "description": "Internal server error",
+            "examples": {
+              "application/json": {
+                "errors": [
+                  {
+                    "errorCode": "internalServerError",
+                    "errorMessage": "An unexpected error has prevented the operation"
+                  }
+                ]
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "503": {
+            "description": "The service is currently unavailable",
+            "examples": {
+              "application/json": {
+                "errors": [
+                  {
+                    "errorCode": "entityOffline",
+                    "errorMessage": "Entity is offline, please try again later."
+                  }
+                ]
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
     }
   },
   "definitions": {
+    "AgencyProposal": {
+      "type": "object",
+      "required": [
+        "agencyProposalEntries",
+        "booked",
+        "importDate"
+      ],
+      "properties": {
+        "agencyProposalEntries": {
+          "type": "array",
+          "description": "List of the proposal entries",
+          "items": {
+            "$ref": "#/definitions/AgencyProposalEntry"
+          }
+        },
+        "agencyProposalType": {
+          "type": "string",
+          "description": "The agency proposal type - equal API for generic proposition",
+          "enum": [
+            "CWTF",
+            "AEBT",
+            "API",
+            "NONE"
+          ]
+        },
+        "approvalLimitDate": {
+          "type": "string",
+          "description": "The date by which the Request must be approved. This element appears only when integrated with Concur Travel"
+        },
+        "autoSelect": {
+          "type": "boolean",
+          "example": false,
+          "description": "The option AutoSelect allows to choose automatically a Proposal"
+        },
```

Comment:
Remove autoSelect section (is not managed today, and no business case yet to have it managed within the agency proposal via v4 API)